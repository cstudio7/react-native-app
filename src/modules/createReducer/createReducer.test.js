import createReducer from './createReducer';

it('creates reducers correctly', () => {
  const initialState = [1];
  const handler = (state, action) => [...state, ...action.payload.data];
  const reducer = createReducer(initialState, { MY_TEST_ACTION: handler });
  expect(
    reducer(initialState, {
      type: 'MY_TEST_ACTION',
      payload: { data: [2, 3] }
    })
  ).toMatchSnapshot();
});
