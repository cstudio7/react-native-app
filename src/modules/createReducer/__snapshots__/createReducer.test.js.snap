// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`creates reducers correctly 1`] = `
Array [
  1,
  2,
  3,
]
`;
